SEUNAGBEYE/React-Redux-Es6
# #161303162 App structure

### What does this PR do?
Setup the App structure
### Description of Task to be completed?
add app component
add about page
add header component
add styles
add routes


### How should this be manually tested?
fetch this branch: `git fetch origin ft-initial-app-structure-161303162`
install all dependencies with `npm i`
start the app `npm start -s` to see the homepage, aboutpage, and the courses page skeleton

### Any background context you want to provide?
NA


### What are the relevant pivotal tracker stories?
[#161303162](https://www.pivotaltracker.com/n/projects/2200608/stories/161303162)

### Screenshots (if appropriate)
![screen shot 2018-10-20 at 4 16 46 pm](https://user-images.githubusercontent.com/25517388/47257270-a17e1880-d483-11e8-88fe-bec7102b8de7.png)
![screen shot 2018-10-20 at 4 16 53 pm](https://user-images.githubusercontent.com/25517388/47257271-a216af00-d483-11e8-888c-10d977939c60.png)
![screen shot 2018-10-20 at 4 16 59 pm](https://user-images.githubusercontent.com/25517388/47257272-a216af00-d483-11e8-8338-7579c0ff9762.png)
